From 4c03a41b9a50ee46490f631731e6a9bed60c0ad8 Mon Sep 17 00:00:00 2001
From: Jamie Lennox <jamielennox@redhat.com>
Date: Mon, 28 Jul 2014 11:16:00 +1000
Subject: [PATCH 2/4] Un-pin requirements

This has previously been submitted upstream and has since been reverted.
Pinning versions is not acceptable for distribution.

Removing coverage altogether as it shouldn't be needed for building.
---
 requirements.txt      |  2 +-
 test-requirements.txt | 11 +++++------
 2 files changed, 6 insertions(+), 7 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index e4d837f..a42590b 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1 +1 @@
-urllib3==1.7.1
+urllib3
diff --git a/test-requirements.txt b/test-requirements.txt
index ceebf1a..4f2cfaa 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -1,12 +1,11 @@
 # test runner
-coverage==3.7.1
-nose==1.3.0
+nose
 
 # testing utilities
-mock==1.0.1
-sure==1.2.3
+mock
+sure
 
 # external frameworks tested against
-httplib2==0.8
+httplib2
 requests
-tornado==3.2
+tornado
-- 
1.9.3

